src/test/com/constructions/Calculator_ternary.java

```diff
@@ -0,0 +1,24 @@
+package test.com.constructions;
+
+import java.util.Scanner;
+
+/**
+ * Умовна конструкція з тернарним оператором:
+ * Створити у класі Calculator метод calculateTernary(int number)
+ * який повинен робити те ж саме що метод calculate але при
+ * цьому використовувати тернарний умовний оператор.
+ * Приклад: 1 - “Odd number: 1”, 2 - “Even number: 2”.
+ */
+
+public class Calculator_ternary {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int num = sc.nextInt();
+
+        System.out.println(calculate(num));
+    }
+
+    static public String calculate(int number) {
```

Comment:
shouldn't be static, create an instance of the object your main method is inside of.